### ISSUE TITLE
--device /dev/snd 'error gathering device information while adding custom device "/dev/snd"'

### ISSUE THREAD

**Issue by ricardoseriani** [docker/docker#12694]
When I try to run add a sound device of my host to my docker container I have the error:

```
docker run -it --rm --device /dev/snd ubuntu
FATA[0000] Error response from daemon: Cannot start container 709637ad0e5d02baa94148f0af7ea4daaa109ad9ce831044ad2cfb29b5f98911: error gathering device information while adding custom device "/dev/snd":not a device node
```

The only way I can have my sound works is mapping each device inside of my /dev/snd using --device and using a --privileged flag

```
docker run -it --rm --device /dev/snd/controlC0 --privileged ubuntu
```

Info:

```
OS: Ubuntu Trusty 14.04.02
Linux shadow 3.16.0-34-generic #47~14.04.1-Ubuntu SMP Fri Apr 10 17:49:16 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
```
```
docker version

Client version: 1.6.0
Client API version: 1.18
Go version (client): go1.4.2
Git commit (client): 4749651
OS/Arch (client): linux/amd64
Server version: 1.6.0
Server API version: 1.18
Go version (server): go1.4.2
Git commit (server): 4749651
OS/Arch (server): linux/amd64
```

```
docker info

Containers: 0
Images: 886
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 929
 Dirperm1 Supported: true
Execution Driver: native-0.2
Kernel Version: 3.16.0-34-generic
Operating System: Ubuntu 14.04.2 LTS
CPUs: 4
Total Memory: 7.696 GiB
Name: shadow
ID: ZIKB:GRCM:MZIA:7OMM:T4CK:GWR3:V4MC:KCRR:3V3U:ZXT7:MMHG:C2XG
WARNING: No swap limit support`

**Comment by LK4D4** [comment]
@ricardoseriani Works for me on master.

**Comment by tianon** [comment]
Indeed, #10745 didn't make it into 1.6.0, so this is only available from either master builds (https://master.dockerproject.com/), or waiting for 1.7.0.

**Comment by LK4D4** [comment]
So, we can close this.
Fixed by #10745

**Comment by jfrazelle** [comment]
yay for 1.7.0 :D

**Comment by ricardoseriani** [comment]
My mistake..
I thought that it was included in 1.6.0
